vendor.conf

```diff
@@ -44,7 +44,7 @@ github.com/containerd/ttrpc 92c8520ef9f86600c650dd540266a007bf03670f
 github.com/containerd/go-runc 9007c2405372fe28918845901a3276c0915689a1
 github.com/containerd/fifo bda0ff6ed73c67bfb5e62bc9c697f146b7fd7f13
 github.com/containerd/continuity f2a389ac0a02ce21c09edd7344677a601970f41c
-github.com/containerd/containerd 59a625defb21c958c25424fa5cc806167e22382e
+github.com/containerd/containerd ed16170c4c399c57f25d6aa1e97b345ed6ab96cb
 github.com/containerd/console 0650fd9eeb50bab4fc99dceb9f2e14cf58f36e7f
 github.com/containerd/cgroups c4b9ac5c7601384c965b9646fc515884e091ebb9
```

Comment:
You have to update this too right?

Reply:
not yet

Reply:
I'll update this reference from containerd then i'll update cri and cgroups in the containerd core and I think we are golden

Reply:
@Random-Liu - How does the ci/cd build work. Does it come from containerd/containerd or does it build from here? Seems like it builds from here in which case we need the updated vendor tree here for ci/cd builds but we need it updated in containerd/containerd for official builds. Or am I 100% crazy...?

Reply:
Yeah, all tests build from here, so we do need to update the dependencies here right now.

Reply:
Currently the cgroup commit in containerd and cri is in sync.  It should be fine for now.